plugins/osdn/podinfoserver.go

```diff
@@ -0,0 +1,138 @@
+package osdn
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+	"net/http"
+	"os"
+	"path"
+	"syscall"
+
+	"github.com/gorilla/mux"
+
+	"github.com/openshift/openshift-sdn/pkg/netutils"
+	"github.com/openshift/openshift-sdn/plugins/osdn/api"
+)
+
+type PodInfoServer struct {
```

Comment:
Fair enough; originally it was just the pod info stuff and I was going to write the cluster info to /etc/cni/net.d/openshift-sdn.conf.  But it turns out kubelet's CNI plugin needs that file immediately at startup, and we don't have cluster info until later so I couldn't do that.  I'll fix this up.